[P9] 10085-Pharo-does-not-work-on-network-drive-on-Windows

tesonep opened the pull request:
Adding support for UNC Network paths.
They are required to correctly support window shares (SMB).

tesonep committed:
Adding support for UNC Network paths.
They are required to correctly support window shares (SMB).

guillep commented on src/FileSystem-Path/Path.class.st:
```diff
@@ -117,7 +117,10 @@ Path class >> from: aString delimiter: aDelimiterCharacter [
+		ifTrue: [ 
+			((aString size >= 2) and: [ aString second = aDelimiterCharacter ])
+				ifTrue: [ NetworkPath ]
+				ifFalse: [AbsolutePath] ]
```
Maybe for a next iteration, but, shouldn't this be dispatched back to the store? Because the NetworkPath is a windows concern, right?

guillep commented on src/FileSystem-Path/Path.class.st:
```diff
@@ -137,12 +140,15 @@ Path class >> isAbsoluteUnixPath: aString [
+	^ (aString size > 2 and: [ aString first isLetter ])
+		  ifTrue: [ "e.g. C:\..." 
+		  aString second = $: and: [ aString third = $\ ] ]
+		  ifFalse: [ false ]
```
Same, shouldn't this be dispatched to the store?

tesonep replied:
Yes, that can be nicer. I will do that change in P10.

tesonep committed:
- Renaming NetworkPath to UNCNetworkPath 
- Fixing transformation of ZnURL in MC to a correct absolute path.
- Extending the time of NoUnusedVariablesLeftTest

tesonep committed:
Handling correctly when the path is relative
